pkg/client/client.go

```diff
@@ -179,11 +193,16 @@ func New(options ...Option) (Client, error) {
 		optionSet.httpClient = http.DefaultClient
 	}
 
+	if optionSet.userAgent == "" {
+		optionSet.userAgent = fmt.Sprintf("go-anxcloud / %s (%s)", version, runtime.GOOS)
```

Comment:
There should typicalally be no space around the slash

Reply:
Good catch :+1: